MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -0,0 +1,530 @@
+        </imageEditor:ImageEditorToolbarSettings>
+    </imageEditor:SfImageEditor.ToolbarSettings>
+</imageEditor:SfImageEditor>
+
+      
+{% endhighlight %}
+
+{% highlight C# %}
+
+    SfImageEditor imageEditor = new SfImageEditor();
+    imageEditor.Source = ImageSource.FromFile("image4.png");
+    imageEditor.ToolbarSettings.ColorPalette.Add(Colors.Yellow);
+    imageEditor.ToolbarSettings.ColorPalette.Add(Colors.Pink);
+    imageEditor.ToolbarSettings.ColorPalette.Add(Colors.Violet);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## ToolbarHeight Customization
+
+You have the flexibility to adjust the toolbar's height, color scheme, orientation, and even create custom toolbars.
+
+### Customize toolbar height 
+
+The toolbar items will be resized based on the height. To change the height of the toolbar, refer to the following code snippet.
+
+{% highlight C# %}
+
+    SfImageEditor imageEditor = new SfImageEditor();
+    imageEditor.Source = ImageSource.FromFile("image4.png");
+    imageEditor.Toolbars[0].Size = 80;
+    imageEditor.Toolbars[1].Size = 50;
+
+{% endhighlight %}
+
+### Toolbar Item Color Customization
+
+You can arrange the toolbar items based on the toolbar the following properties:
+
+1.  `DisabledColor` -  Color for disabled items.
+2.  `TextColor`  - Color for text within items.
+3.  `IconColor` - Color for icons within items.
+
+To change the toolbar item Text and Icon color, refer to the following code snippet.
+
+{% tabs %}
+
+{% highlight XAML %}
+
+<imageEditor:SfImageEditor Source="table.png">
+    <imageEditor:SfImageEditor.ToolbarSettings>
+        <imageEditor:ImageEditorToolbarSettings DisabledColor="Violet" TextColor="pink" IconColor="Yellow" />
+    </imageEditor:SfImageEditor.ToolbarSettings>
+</imageEditor:SfImageEditor>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image4.png");
+imageEditor.ToolbarSettings.DisabledColor = Colors.Violet;
+imageEditor.ToolbarSettings.TextColor = Colors.Pink;
+imageEditor.ToolbarSettings.IconColor = Colors.Yellow;
+    
+{% endhighlight %}
+
+{% endtabs %}
+
+## Toolbar color customization
+
+Border and background color of the toolbar can be customized with the help of `Stroke` and `BackGround` properties in `ImageEditorToolbarSettings`.
+
+{% tabs %}
+
+{% highlight XAML %}
+
+<imageEditor:SfImageEditor Source="table.png">
+    <imageEditor:SfImageEditor.ToolbarSettings>
+        <imageEditor:ImageEditorToolbarSettings Stroke="Violet" Background="pink" />
+    </imageEditor:SfImageEditor.ToolbarSettings>
+</imageEditor:SfImageEditor>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image4.png");
+imageEditor.ToolbarSettings.Stroke = Colors.Violet;
+imageEditor.ToolbarSettings.Background = Colors.Pink;
+    
+{% endhighlight %}
+
+{% endtabs %}
+
+## Toolbar Orientation and Position
```

Comment:
add separate topics for position and orientation'

Reply:
Added separate topics for position and Orientation